htdocs/index.php

```diff
@@ -98,6 +98,16 @@
 
 print load_fiche_titre('&nbsp;', $resultboxes['selectboxlist'], '', 0, '', 'titleforhome');
 
+if (DOL_VERSION  != $conf->global->MAIN_VERSION_LAST_UPGRADE) {
```

Comment:
This should not be required anymore.
We have this in current main.inc.php

```
// If an upgrade process is required, we call the install page.
$checkifupgraderequired = false;
if (getDolGlobalString('MAIN_VERSION_LAST_UPGRADE') && getDolGlobalString('MAIN_VERSION_LAST_UPGRADE') != DOL_VERSION) {
	$checkifupgraderequired = true;
}
if (!getDolGlobalString('MAIN_VERSION_LAST_UPGRADE') && getDolGlobalString('MAIN_VERSION_LAST_INSTALL') && getDolGlobalString('MAIN_VERSION_LAST_INSTALL') != DOL_VERSION) {
	$checkifupgraderequired = true;
}
if ($checkifupgraderequired) {...
```

and if upgrade is required, a redirect to the upgrade process is automatically done. No choice.

Reply:
Yes, that doesn't seem necessary any more. The issue should have been closed a long time ago.